apps/cookbook/src/app/examples/card/card-clickable-example/card-clickable-example.component.html

```diff
@@ -0,0 +1,14 @@
+<kirby-card (click)="function()">
+  <kirby-card-header title="Clickable"></kirby-card-header>
+  <div class="card-content">
+    <p>
+      Giving a card any click event <code>(click)="function()"</code> will change your cursor to a
+      pointer when hovering the card. There is also added accessibility so that you can focus the
+      card by tabbing and clicking with either space or enter.
```

Comment:
I would suggest something like this just to be a bit more specific on the behavior: 
```suggestion
      pointer when hovering the card. To ensure that the card is accessible by keyboard, a tab-index is added so the card can be focused by tabbing. To mirror a buttons behavior, any registered click-event will also be called by pressing enter and space when the card is focused. 
```